src/components/Nav/NavComponent/NavTop/NavTop.scss

```
@@ -0,0 +1,31 @@
+@import '../../../../styles/variables.scss';
+
+.navTop {
+  list-style-type: disc;
+  @include flex(space-between, center);
+  height: 46px;
+  .navUl {
+    height: 46px;
+    display: flex;
+    align-items: center;
+
+    .list {
+      color: #c0c0c0;
+      font-size: 12px;
+      line-height: 18px;
+      font-weight: 500;
+      padding-right: 9px;
+
+      a,
+      a:hover,
+      a:focus,
+      a:active {
+        text-decoration: none;
+        color: #c0c0c0;
+      }
+    }
+    .list:last-child {
+      padding-right: 0px;
+    }
```

```suggestion
    .list {
      color: #c0c0c0;
      font-size: 12px;
      line-height: 18px;
      font-weight: 500;
      padding-right: 9px;
      
         &:last-child {
         padding-right: 0px;
        }

      a{
      text-decoration:none;
         &:hover{}
         &:focus{}
         &:active{}
      }

    }
    
```

Sass에서는 
&(ampersand) , parent selector 를 활용해서 가상선택자나 가상클래스를 사용합니다.